src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -54,8 +54,17 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
      */
     public static Set<Tag> getTagSet(String... strings) {
         return Arrays.stream(strings)
-                .map(Tag::new)
+                .map(tagName -> {
+                    if (PersonTagType.isValidPersonTag(tagName)) {
+                        return new PersonTag(tagName);
+                    } else if (PropertyTagType.isValidPropertyTag(tagName)) {
+                        return new PropertyTag(tagName);
+                    } else {
+                        return new Tag(tagName);
```

Comment:
Maybe should consider using a helper/util function for this since it is repeated?